stdlib/decimal.pyi

```diff
@@ -35,6 +36,16 @@ _TrapType: TypeAlias = type[DecimalException]
 # At runtime, these classes are implemented in C as part of "_decimal".
 # However, they consider themselves to live in "decimal", so we'll put them here.
 
+# This type isn't exposed at runtime. It calls itself decimal.ContextManager
+@final
+@type_check_only
+class _ContextManager:
+    new_context: Context
+    saved_context: Context
```

Comment:
did we just make up these attributes, I wonder? I can't find them on the class or instances of the class at runtime:

```py
>>> import decimal
>>> e = decimal.localcontext()
>>> e.new_context
Traceback (most recent call last):
  File "<python-input-4>", line 1, in <module>
    e.new_context
AttributeError: 'decimal.ContextManager' object has no attribute 'new_context'
>>> e.saved_context
Traceback (most recent call last):
  File "<python-input-5>", line 1, in <module>
    e.saved_context
AttributeError: 'decimal.ContextManager' object has no attribute 'saved_context'
```

Reply:
Not totally made up. I checked the blame and they're a holdover from 2.7.

Reply:
Yes, I just came to the same conclusion!

```pycon
Python 2.7.18 (default, Nov 17 2024, 00:12:35) 
[GCC Apple LLVM 15.0.0 (clang-1500.3.9.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import decimal
>>> d = decimal.localcontext()
>>> d.new_context
Context(prec=28, rounding=ROUND_HALF_EVEN, Emin=-999999999, Emax=999999999, capitals=1, flags=[], traps=[Overflow, DivisionByZero, InvalidOperation])
```

Reply:
Let's get rid of them while we're here; Python 2.7 is long dead!

```suggestion
```